### ISSUE TITLE
Keep3r Jobs Migration + Ongoing Maintenance

### ISSUE THREAD

**Issue by vPEPO** [yearn/ychad-audit#64]
### Scope

Migration of Yearn jobs from KP3R v1 to v2 + ongoing maintenance that includes:
- LP credits management
- alerts for potential job errors
- MEV monitoring 
- job performance monitoring (check if keepers are working yearn’s jobs, and if not, work them with our own keeper)

### Plan

Migrating Keep3r Jobs from V1 to V2 is the main priority and will be done ASAP. 

The ongoing tasks will be performed for 12 months from the starting date.

### Deadline

6/6/2023

### People

[DeFi Wonderland](https://twitter.com/DeFi_Wonderland)

### Money

The budget requested will be spent on the development of the jobs and ongoing support/monitoring for the jobs developed by us.

### Amount

435000 DAI

### Wallet address

0x45fEEBbd5Cf86dF61be8F81025E22Ae07a07cB23

### Reporting

- [X] I agree to post weekly updates

**Comment by banteg** [comment]
Thanks for filling out the request. Would you be open to receive this as a stream?

**Comment by vPEPO** [comment]
> Thanks for filling out the request. Would you be open to receive this as a stream?

Yes, we've already reviewed and experimented with LlamaPay ourselves, so we are confident about the security of the product

**Comment by banteg** [comment]
Stream of 36,250 DAI/month created:
- https://ethtx.info/mainnet/0x4fa67cd76454147ef697670ecfb6a1d5cc6077ab8f248ee9361d1d5f7dd19642/
- https://llamapay.io/streams?chainId=1&address=0x45feebbd5cf86df61be8f81025e22ae07a07cb23

**Comment by vPEPO** [comment]
To keep this updated: The stream was modified by our request to 0x8bA72884984f669aBBc9a5a7b441AD8E3D9a4fD3

https://llamapay.io/salaries/Ethereum/0x8bA72884984f669aBBc9a5a7b441AD8E3D9a4fD3

**Comment by 0xGorilla** [comment]
A new stream has been agreed upon:
**Start date:** June 8th 2023
**Duration:** 6 months
**Amount:** 15,000 DAI per month

**Comment by 0xGorilla** [comment]
The new stream should be paid to 0xf58e95124d655731664c509C07E733348d7081c0
